helpers/dmesg.py

```diff
@@ -114,3 +114,20 @@ def wait_for_msg(msg, timeout, permissive):
         # otherwise we either spend too much time on timeouts or observe
         # spurious exceptions.
         tf_cfg.dbg(2, 'No "%s" log record and no ratelimiting' % msg)
+
+
+def unlimited_rate_on_tempesta_node(func):
+    """
+    The decorator turns off dmesg messages rate limiting to ensure important
+    messages are caught.
+    """
+    def func_wrapper(*args, **kwargs):
+        get_cost_cmd = 'cat /proc/sys/net/core/message_cost'
+        msg_cost = remote.tempesta.run_cmd(get_cost_cmd)[0].strip()
+        remote.tempesta.run_cmd('echo 0 > /proc/sys/net/core/message_cost')
+        res = func(*args, **kwargs)
```

Comment:
The wrapper stores and restores the message cost for each particular test. It works great till test passes. But if the test fails an exception will be thrown and the wrapper will be also aborted, isn't it? If so the message cost wouldn't be restored to initial value for failed tests, causing the the next tests to have unlimited rate limits. 

Need to catch exceptions, restore the message cost and re-throw the exception. Also note, If Tempesta has crashed during `func()`, restoring the message cost will also generate the exception.

Reply:
Updated the code to use try-finally.
Initial message cost is obtained without exception protection, but I think that's fine. If we fail to communicate with the remote node, it's better to fail earlier.

Reply:
Yeah, totally agree here.